tests/hooks_test.py

```diff
@@ -145,3 +145,24 @@ def test_python_rst_backticks_positive(s):
 )
 def test_python_rst_backticks_negative(s):
     assert not HOOKS['rst-backticks'].search(s)
+
+
+@pytest.mark.parametrize(
+    's',
+    (
+        str(b'\x80abc', errors='replace'),
+    ),
+)
+def test_text_unicode_replacement_char_positive(s):
+    assert HOOKS['text-unicode-replacement-char'].search(s)
+
+
+@pytest.mark.parametrize(
+    's',
+    (
+        str(b'\x80abc', errors='backslashreplace'),
```

Comment:
probably just inline `r'\x80abc'`

Reply:
Opted to do that without the `r`, not that it would make any difference here whether the char ends up as the char `\x80` or the string `\\x80`, as it just represents a string without those replacement chars.

Come to think of it, I'll just remove that altogether as it's potentially misleading as witnessed here.